docs/migration-2-to-3.md

```diff
@@ -114,4 +114,15 @@ If you define this bean with custom configuration you need to update your config
 ```
 
 You can check the new API
-definition [here](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/s3/S3Client.html).
\ No newline at end of file
+definition [here](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/s3/S3Client.html).
+
+## Kafka
+
+There has been introduced a new default value for the `sda.kafka.consumer.dlt.pattern` property,
+which is `dlt-<topic>`. If you want to keep the old behaviour, which is appending `.DLT` to the
+consumed topic, you have to
+explicitly unset the property.
```

Comment:
Could you give a practical example? It is not clear to me what would be unsetting the property.